decision-records/001-management-service.md

```diff
@@ -0,0 +1,143 @@
+# Typerighter rule management service architecture
+
+## Status
+
+Proposed
+
+## Context
+
+Typerighter requires a rule management system to serve rule owners (RO) (those managing the corpus of rules) and rule users (RU) (those receiving matches via the Typerighter service). As a summary, key user stories include:
+
+-   Browsing, sorting, and filtering rules (RO)
+
+-   CRUD for rules, including the ability to evaluate rules in a draft context, before applying them to the published corpus (RO)
+
+-   Understanding the history of a rule, including seeing when and what was changed by whom (RO, RU)
+
+-   Adding feedback to rules based on match performance (RU)
+
+See the [trello board](https://trello.com/b/PlsiUqYq/typerighter) for a comprehensive list.
```

Comment:
Maybe we could copy the relevant list from Trello to another doc in this repo, so that it persists, as the board is likely to change when we do another wave of work.

Reply:
Great idea 👍